docs/references/openapi.yaml

```diff
@@ -933,6 +958,161 @@ paths:
           description: The file provided was not found. Check that the file was processed and that you used it in a signature request.
       tags:
         - Signing
+  /request/status:
+    post:
+      operationId: request_status
+      summary: Show status of signature request
+      parameters:
+        - name: request_url
+          in: query
+          required: true
+          schema:
+            type: string
+            format: url
+          description: The URL of the signature request to check.
+        - name: password
+          in: query
+          schema:
+            type: string
+          description: Password for password encrypted requests
+      responses:
+        200:
+          description: All pending signature requests for the file were cancelled.
```

Comment:
We return 200 when all the requests were cancelled? Same below for /request/file/status. Can you explain?

Reply:
Copy and paste issue from the place where it already was copy paste issue :D